content/blog/2020-05-08-dvc-ambassador-program.md.md

```diff
@@ -0,0 +1,151 @@
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+descriptionLong: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+picture: 2020-05-08/Ambassador_Header.png
+author: marcel_rd
+commentsUrl: https://discuss.dvc.org/t/new-video-reimagining-devops-for-machine-learning/341
+tags:
+  - Ambassador
+  - Volunteer
+  - Talks
+  - Meetup
+  - Outreach
+---
+
+Software can be everywhere, but developers can’t - that’s why ambassadors, folks
+who do outreach and community building around projects they love, are a key part
+of the open-source community. DVC is starting an ambassador program to help
+people who are passionate about our mission get involved.
+
+What is that mission, exactly? DVC is founded on the principle of engineering
+solutions for making data science and machine learning rigorous and
+reproducible. If this matters to you, too, you might be a good fit for our
+ambassador program!
+
+As an ambassador, you’ll act as a bridge between DVC in your community. There
+are lots of ways to do this, big and small. For example:
+
+- Write a blog post talking about how you use DVC in your projects
+- What about creating a network of DVC users and data scientists in your town?
+  Even though we’re self-isolating now, you can still organize online meetups.
+  [We already did one!](https://tulu.la/events/dvc-virtual-meetup-2020-00032c).
+  We help cover costs to organize meetups.
+- Do you want to talk about DVC at your office, or at a conference? We help
+  speakers develop talks, and we have some discretionary funds for travel on a
+  case-by-case basis.
+- Want to develop a feature for DVC? We welcome contributions to the code base,
+  even if it’s your first pull request ever.
+
+Being an ambassador means getting closer to the team in charge of DVC, but at
+the same time, it means going farther to reach people outside the organization-
+including people who don’t know about DVC yet, people who need some help getting
+started, and people who are already excited about our mission and want to find
+meaningful ways to pitch in.
+
+It’s challenging, no doubt, but being an ambassador can be an amazing
+experience! Before becoming DVC’s first official ambassador, I served as a
+[Fedora ambassador](https://fedoraproject.org/wiki/User:Mribeirodantas) for
+several years. I got a lot out of it: I participated in lots of events, made
+friends, and traveled. Every single time I’ve contributed to an open source
+project, I got this nice feeling that it was all worth it. If you agree with the
+core values of the project, a great relationship lies ahead :).
+
+## About Iterative and DVC
+
+DVC got started in 2017 as a personal project by Dmitry Petrov (on May 4,
+[we celebrated our 3rd birthday](https://dvc.org/blog/dvc-3-years-and-1-0-release)).
+In 2018, Dmitry teamed up with his co-founder Ivan Shchenklein to incorporate
```

Comment:
`co-founder Ivan Shchenklein...` more info about Ivan might also help. Users should see we are strong folks :)
For example, Ivan has experience with a startup - he co-founded https://tweetedtimes.com/ startup. He is an open-source "veteran" - one of the core contributors to https://www.sedna.org/ which is a document-oriented database like MongoDB but before it.

Reply:
Woah, that's awesome! I also think this is very important.